run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cd `dirname $0`
+cd "$(dirname "$0")"
```

Comment:
Apparently Solaris 11's `/bin/sh` is `ksh93`: https://blogs.oracle.com/OTNGarage/entry/new_shell_in_oracle_solaris